src/Pages/shop/Components/product/Product.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "../../../../Styles/common.scss";
+import "./Product.scss";
+
+class Product extends Component {
+  render() {
+    return (
+      <div className="product__wrapper">
```

Comment:
```suggestion
      <div className="Product">
```
컴포넌트 최상위 태그의 클래스명은 컴포넌트명과 일치시켜주세요